### ISSUE TITLE
Last version of the plugin does not install

### ISSUE THREAD

**Issue by dlvenable** [awslabs/logstash-output-amazon_es#174]
The latest version of the `amazon_es` plugin is 7.0.1. And the latest on the 6.4 series is 6.4.2. However, Logstash consistently installs 6.4.1.

Full list of Gems: https://rubygems.org/gems/logstash-output-amazon_es/versions

**Steps to Reproduce**

```
bin/logstash-plugin install logstash-output-amazon_es
```

Output:

```
Using JAVA_HOME defined java: /Library/Java/JavaVirtualMachines/amazon-corretto-11.jdk/Contents/Home
WARNING, using JAVA_HOME while Logstash distribution comes with a bundled JDK
OpenJDK 64-Bit Server VM warning: Option UseConcMarkSweepGC was deprecated in version 9.0 and will likely be removed in a future release.
Validating logstash-output-amazon_es
Found logstash-output-amazon_es (6.4.1), but was for platform x86_64-linux
Installing logstash-output-amazon_es
/Users/dlv/Documents/Development/elastic/logstash-7.14.2/vendor/bundle/jruby/2.5.0/gems/bundler-1.17.3/lib/bundler/rubygems_integration.rb:200: warning: constant Gem::ConfigMap is deprecated
Installation successful
```

**Comment by dlvenable** [comment]
I yanked the `x86_64` platform for version 6.4.1.

```
gem yank --version 6.4.1 --platform x86_64-linux logstash-output-amazon_es
```

**Comment by dlvenable** [comment]
Validating on a clean Logstash 7.16.1 installation:

Install:

```
 bin/logstash-plugin install logstash-output-amazon_es
```
```
Using bundled JDK: /home/ec2-user/elastic/logstash-7.16.1/jdk
OpenJDK 64-Bit Server VM warning: Option UseConcMarkSweepGC was deprecated in version 9.0 and will likely be removed in a future release.
Validating logstash-output-amazon_es
Resolving mixin dependencies
Installing logstash-output-amazon_es
Installation successful
```

Check version:

```
bin/logstash-plugin list --verbose logstash-output-amazon_es
```
```
Using bundled JDK: /home/ec2-user/elastic/logstash-7.16.1/jdk
OpenJDK 64-Bit Server VM warning: Option UseConcMarkSweepGC was deprecated in version 9.0 and will likely be removed in a future release.
logstash-output-amazon_es (7.0.1)
```

**Comment by dlvenable** [comment]
The latest version for download is now 7.0.1.

Advisory: If you have been using this plugin, you may have been inadvertently getting version 6.4.1.

If you still need 6.4.1, you can install via the following command:

```
bin/logstash-plugin install --version 6.4.1 logstash-output-amazon_es
```

**Comment by dlvenable** [comment]
@nosnilmot , Unfortunately, I'm unsure if I can determine which commits to tag.

These releases were created months ago and not tagged at the time. I believe I'd only be able to guess at this point.

**Comment by nosnilmot** [comment]
> @nosnilmot , Unfortunately, I'm unsure if I can determine which commits to tag.
> 
> These releases were created months ago and not tagged at the time. I believe I'd only be able to guess at this point.

Perhaps it would be safest to create a new, tagged, release 7.0.2 then?

I had resorted to building from git yesterday (before I understood that '--no-verify' was able to get around #173), but was left with little option than to build HEAD in the absence of recent tags (and even less ability to guess at a suitable revision than you), and that's still the case should anything like this come up again.

**Comment by dlvenable** [comment]
@nosnilmot , We released version 7.1.0 and tagged this release. This should help if another need arises to build from source.
